internal/plugins/helm/v1/init.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/util/validation"
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+	"sigs.k8s.io/kubebuilder/pkg/plugin"
+	"sigs.k8s.io/kubebuilder/pkg/plugin/scaffold"
+
+	"github.com/operator-framework/operator-sdk/internal/kubebuilder/cmdutil"
+	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/chartutil"
+	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/scaffolds"
+	"github.com/operator-framework/operator-sdk/internal/plugins/sdk"
+)
+
+type initPlugin struct {
+	config        *config.Config
+	apiPlugin     createAPIPlugin
```

Comment:
I'm not a fan of embedding one plugin's struct in another, but this is internal so is ok (and can be removed more easily).

Reply:
It was preference made by @joelanford

Reply:
I chose to do it this way to keep things DRY. Otherwise we'd repeat a bunch of code between the plugins since new/init (currently) supports initializing a new project and creating an API all in a single command.